piqueserver/core_commands/movement.py

```diff
@@ -74,6 +79,9 @@ def do_move(connection, args, silent=False):
         player = connection.name
     # player specified
     elif arg_count == 2 or arg_count == 4:
+        if not connection.rights.admin or connection.rights.move_others:
```

Comment:
Shouldn't this be `if not (connection.rights.admin or connection.rights.move_others):`

Reply:
good catch yes

Reply:
```suggestion
        if not (connection.rights.admin or connection.rights.move_others):
```